Cheatsheet.md

```diff
@@ -0,0 +1,554 @@
+# Cheatsheet
+### Commonly used [array](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Arrays) functions
+
+## [1. map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) 

```

Comment:
make all the functions to header 3 instead of header 2.

Reply:
When I change it to header 3 the headings disappear in the preview.

Reply:
Make it change as it doesn't look good that the main header is smaller that the sub header.

Reply:
Would same size be OK ?

Reply:
Same size would be little bit off.
Let's ask @alodahl regarding this.

Reply:
Done.